example-apps/my-nest-project/src/app.module.ts

```diff
@@ -10,9 +10,10 @@ import { AppService } from './app.service';
 import { IsAuthor } from './is-author.condition';
 import { AuthnGuard } from './authn.guard';
 import { InMemoryPermissionProvider } from './in-memory-permission.provider';
+import { UsersModule } from './users/users.module';
 
 @Module({
-  imports: [AuthorizationModule],
+  imports: [AuthorizationModule.use(), UsersModule],
```

Comment:
why is this necessary? Don't know of any other Module which does this. Is this really necessary?

Reply:
I'll check again

Reply:
Removed `use` method, instead we could rely on re-exporting in userland instead. Do you prefer that?

Reply:
I don't understand consequences, let's discuss in person